buildsystem/dependencies.gradle

```diff
@@ -0,0 +1,55 @@
+allprojects {
+    repositories {
+        google()
+        jcenter()
+    }
+}
+
+task clean(type: Delete) {
+    delete rootProject.buildDir
+}
+
+ext {
+    // Jetbrains
+    kotlin_version = '1.3.50'
+
+    // AndroidX
+    gradle_version = '3.5.2'
+    androidx_version = '1.1.0'
+    constraintlayout_version = '1.1.3'
+
+    // Test
+    junit_version = '4.12'
+
+    // AndroidTest
+    androidx_junit_version = "1.1.1"
+    espresso_core_version = "3.2.0"
+
+    versionNumbers = [
+            compileSdkVersion   : 29,
+            buildToolsVersion   : "29.0.2",
+            minSdkVersion       : 19,
+            targetSdkVersion    : 29,
+    ]
+
+    classpathDependencies = [
+            androidToolsBuild       : "com.android.tools.build:gradle:${gradle_version}",
+            kotlinGradlePlugin      : "org.jetbrains.kotlin:kotlin-gradle-plugin:${kotlin_version}",
+    ]
+
+    androidXDependencies = [
+            appCompat               : "androidx.appcompat:appcompat:${androidx_version}",
+            coreKtx                 : "androidx.core:core-ktx:${androidx_version}",
+            constraintlayout        : "androidx.constraintlayout:constraintlayout:${constraintlayout_version}",
+    ]
+
+    testDependencies = [
+            jUnit                   : "junit:junit:${junit_version}",
+    ]
+
+    androidTestDependencies = [
+            androidXJUnit           : "androidx.test.ext:junit:${androidx_junit_version}",
+            espressoCore            : "androidx.test.espresso:espresso-core:${espresso_core_version}",
+    ]
+
+}
```

Comment:
Chapeau!

Reply:
Wow! I dind't know we can write the gradle in this way =O

Reply:
Gracias, yo lo aprende de un compañero aqui que es un crack del gradle.
me acabo de dar cuenta que no he puesto el kotlin dependency en dependencies 😓 
Lo actualizare antes de mergear :)